src/app/content/components/Page.tsx

```diff
@@ -45,16 +46,18 @@ export class PageComponent extends Component<PropTypes> {
     const pageContent = cachedPage ? cachedPage.content : '';
 
     return this.props.references.reduce((html, reference) =>
-      html.replace(reference.match, content.getUrl(reference.params))
+      html.replace(reference.match, toRelativeUrl(currentPath, content.getUrl(reference.params)))
     , pageContent)
       // remove body and surrounding content
       .replace(/^[\s\S]*<body.*?>|<\/body>[\s\S]*$/g, '')
       // fix assorted self closing tags
       .replace(/<(em|h3|iframe|span|strong|sub|sup|u)([^>]*?)\/>/g, '<$1$2></$1>')
       // remove page titles from content (they are in the nav)
       .replace(/<h(1|2) data-type="document-title".*?<\/h(1|2)>/, '')
-      // target blank qualified links
+      // target blank and add `rel` to links that begin with: http:// https:// //
       .replace(/<a(.*?href="(https?:\/\/|\/\/).*?)>/g, '<a target="_blank" rel="noopener nofollow"$1>')
+      // same as previous, but allow indexing links to relative content
+      .replace(/<a(.*?href="\.\.\/.*?)>/g, '<a target="_blank"$1>')
```

Comment:
Does this add `target="_blank"` to **all** relative links and rely on the JS click handler to `e.preventDefault()`? If so, that seems a little fragile.

Reply:
it adds the target blank to links to different books (they start with `../`)